### ISSUE TITLE
[accessibility] alt text for custom emojis

### ISSUE THREAD

**Issue by annando** [tootsuite/mastodon#9269]
### Pitch

The custom emojis that are solved as pictures should have the option to enter an alternate text for screen readers.

### Motivation

From the perspective of the accessibility the custom emojis are currently a nightmare since only the shortcode is displayed as alternate text. Having some description would help visually impaired persons that rely on screen readers.

**Comment by ZoeBijl** [comment]
This is a good idea.

**Comment by hibiyasleep** [comment]
we could utilize `iTXt` field of png file (idk on gif but there's should be) for this feature, for

* easy i18n of alt text
* seamless implementation?
* convenience emoji copying

and/or some more reason.

**Comment by arielsbecker** [comment]
So, as far as I can tell, we need to modify these two:

* app/models/custom_emoji.rb
* db/schema.rb

by adding a new string field on the first one to store a screenreader text. Alternatively, it can be an index pointing to an ancillary table that stores all the i18n strings for that particular emoji, but since it applies only to custom emojis (which are mostly valid on a single instance), this might introduce too many complexities.
